src/components/ProfileContent/ProfileContent.module.css

```
@@ -14,3 +14,10 @@
   flex-wrap: wrap;
   gap: 32px;
 }
+.overlayVideo {
+  position: absolute;
+  top: 0;
```

вместо `top: 0; width: 100%; height: 100%;` должно работать `inset: 0`

```suggestion
  inset: 0
```